platforms/web/src/modules/register.ts

```diff
@@ -29,44 +29,45 @@ import { LocalStorageService } from '#src/services/LocalStorageService';
  * })
  * ```
  */
+export const register = async () => {
```

Comment:
Hm, do we need to wrap it in async in case we don't use `rebind`. Or do you have it within your local implementation? I remember we wanted to make it sort of react agnostic and do this binding not in the component.

Reply:
Yes, that is correct. It could be non-async here, but then it needs to be changed when rebinding a service. In our fork, we've swapped some services, so it needs to be async. Do you prefer it to be sync be default?